src/test/kotlin/com/terraformation/backend/customer/AppNotificationServiceTest.kt

```diff
@@ -218,6 +224,48 @@ internal class AppNotificationServiceTest : DatabaseTest(), RunsAsUser {
         "Notification should match that of an accession scheduled to end drying")
   }
 
+  @Test
+  fun `should store nursery seedling batch ready notification`() {
+    // add a second user to check for multiple notifications
+    insertUser(otherUserId)
+    insertOrganizationUser()
+
+    val batchId = BatchId(100)
+    val batchNumber = "22-2-001"
+    val speciesId = SpeciesId(100)
+    val nurseryName = "my nursery"
+
+    insertSpecies(speciesId)
+    insertBatch(
+        BatchesRow(
+            id = batchId,
+            batchNumber = batchNumber,
+            speciesId = speciesId,
+            facilityId = facilityId))
```

Comment:
I don't think it will matter for purposes of this test, but this will be inserting a batch into a seed bank facility. No need to change anything here, just noting it for the future in case at some point we add an integrity constraint at the database level to enforce that the facility type is correct.

Reply:
I can see about creating a separate facility 👍